eland/ml/transformers/xgboost.py

```diff
@@ -58,7 +58,7 @@ def get_feature_id(self, feature_id: str) -> int:
             except ValueError:
                 raise RuntimeError(f"Unable to interpret '{feature_id}'")
         f_id = self._feature_dict.get(feature_id)
-        if f_id:
+        if f_id or f_id == 0:
```

Comment:
would `f_id is not None` work as well? I am not sure which is more pythonic.

Reply:
`fd_id is not None` works too :) Wasn't sure what type of falsy the value could be.